spec/ar2dto/record_to_dto/attributes_spec.rb

```diff
@@ -61,8 +61,8 @@
       it "exposes methods to access the columns set by persistence" do
         expect(subject).to have_attributes(
           id: user.id,
-          created_at: user.created_at,
-          updated_at: user.updated_at
+          created_at: user.created_at.as_json,
```

Comment:
This looks a bit weird to me. Is it returning a datetime as json? 
Can this be avoided?